package.json

```diff
@@ -0,0 +1,25 @@
+{
```

Comment:
You need to git pull the current changes otherwise you will overwrite package.json